{{cookiecutter.repo_name}}/Makefile

```diff
@@ -0,0 +1,46 @@
+.DEFAULT_GOAL := validate
+
+clean:
+	find . -name '*.pyc' -delete
+	coverage erase
+	rm -rf assets coverage
+
+requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+validate_python: clean
+	DISABLE_MIGRATIONS=True coverage run --branch --source={{cookiecutter.repo_name}} ./manage.py test {{cookiecutter.repo_name}} --settings={{cookiecutter.repo_name}}.settings.test
+	coverage report
+	make quality
+
+quality:
+	pep8 --config=.pep8 {{cookiecutter.repo_name}}
+	pylint --rcfile=pylintrc {{cookiecutter.repo_name}}
+
+validate: validate_python
+
+static:
+	python manage.py collectstatic --noinput -v0
+
+html_coverage:
+	coverage html && open htmlcov/index.html
+
+extract_translations:
+	python manage.py makemessages -l en -v1 -d django
+	python manage.py makemessages -l en -v1 -d djangojs
+
+dummy_translations:
+	cd {{cookiecutter.repo_name}} && i18n_tool dummy -v
```

Comment:
Wouldn't it be easier to move the `.tx` directory up one level, since it's about running the working tree, and not part of the installed package, and then we wouldn't need the `cd` commands here, I think?